nixos/tests/kanidm.nix

```diff
@@ -71,5 +85,8 @@ import ./make-test-python.nix ({ pkgs, ... }:
         client.wait_until_succeeds("kanidm login -D anonymous && kanidm self whoami | grep anonymous@${serverDomain}")
         rv, result = server.execute("kanidmd recover_account -c ${serverConfigFile} idm_admin 2>&1 | rg -o '[A-Za-z0-9]{48}'")
         assert rv == 0
+        client.wait_for_unit("kanidm-unixd.service")
+        rv, result = client.execute(" kanidm_unixd_status 2>&1 | rg -o 'working!'")
```

Comment:
1. There’s an extra leading space in the string
2. The `2>&1` is unnecessary?
3. I see that the `-o` flag is used above as well, but I don’t understand why, it should work without
4. Last but not least, I think grep is installed by default, so a simple ` | grep 'working!'` as in line 83 should work as well :)

Reply:
1. :see_no_evil: 
2. It was, only when using the `execute` command
3. & 4 regular grep :ok_hand: